### PR TITLE
fixup panics and properly redirect to slack server

### PR DESCRIPTION
**Pull request by adamdecaf**
I ran into some "cannot read property" from signing up with a slack invite. This change works for me to redirect and not panic on `slack.moov.io`.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -237,9 +237,15 @@ function slackin({
...
           }
-          return res.status(400).json({ msg: inviteErr.message });
+          return res.status(400).json({
+            msg: inviteErr.message,
+            redirectUrl: `https://${org}.slack.com/`,
+          });
         }
-        return res.status(200).json({ msg: 'WOOT. Check your email!' });
+        return res.status(200).json({
+          msg: 'WOOT. Check your email!',
```
Comment (wadearnold) [inline lib/index.js]: Without a `woot` nobody is happy.

**Comment by emedvedev** [comment]
Thanks! Fixed the tests and merged :)

**Comment by adamdecaf** [comment]
Awesome! Thanks for the quick merge. What's the process for a new container release? I've built and run an image myself with this which works for now.

**Comment by emedvedev** [comment]
The containers are built automatically: https://hub.docker.com/repository/docker/emedvedev/slackin-extended. You can just grab the latest master or any active branch from there.

**Pull request merged**